packages/number-input/src/Select/Select.tsx

```diff
@@ -0,0 +1,111 @@
+import React from 'react';
+
+import { ButtonProps } from '@leafygreen-ui/button';
+import { cx } from '@leafygreen-ui/emotion';
+import { useDarkMode } from '@leafygreen-ui/leafygreen-provider';
+import { Option, Select as SelectComponent } from '@leafygreen-ui/select';
+
+import { UnitOption } from '../NumberInput/NumberInput.types';
+import { SelectButton } from '../SelectButton';
+
+import {
+  selectDisabledStyles,
+  selectStyles,
+  wrapperBaseStyles,
+} from './Select.styles';
+import { SelectProps } from './Select.types';
+
+/**
+ * @internal
+ */
+export function Select({
```

Comment:
Should we call this `UnitSelect` to avoid confusion?

Reply:
i'll change this